NitroxModel/DataStructures/NitroxId.cs

```diff
@@ -52,5 +52,15 @@ public override string ToString()
         {
             return guid.ToString();
         }
+
+        public static bool operator ==(NitroxId id1, NitroxId id2)
+        {
+            return id1?.guid == id2?.guid;
+        }
+
+        public static bool operator !=(NitroxId id1, NitroxId id2)
+        {
+            return !(id1 == id2);
+        }
```

Comment:
Is this needed anymore as you are not using/changing the NitroxId anywhere else?

Reply:
The new direction is to use a deterministic NitroxId, so I am changing the class now. I would like to keep the operators as there are lots of places in the code where == is used to check equality.